### PR TITLE
75 Update SDK documentation

### LINKED ISSUE #75: Update SDK documentation
with this PR https://github.com/casper-ecosystem/cep18/pull/69 we brought our CEP 18 contract up to be consistent with CEP78 and added CES events.   (events modality).    We need to follow up with the appropriate documentation updates.

### PR DESCRIPTION
**Pull request by ipopescu**
Closes #75 

The JS SDK documentation in this repo was up-to-date. I have just reformatted it to follow our general documentation guidelines and added some explanations for the error codes. Also, I copied some examples from the tests into the js-client README for easy access.

**Reviewers**
@gyroflaw, please review the tech content, especially the error codes and additional examples copied over (9bcc5574494c9c90049e2377caf52a1f323e8a32 and 9bcc5574494c9c90049e2377caf52a1f323e8a32)
@ACStoneCL, please review the documentation flow (a093f90a51c92d4978c023d4fbdabb5e13f08179 and overall)

Just FYI: @hoffmannjan, @deuszex

### CODE REVIEW

File: client-js/README.md [hunk 1]
```diff
@@ -6,140 +6,220 @@ This JavaScript client gives you an easy way to install and interact with the Ca
...
+      [owner]
+    );
+```
 
-- **Allowance**
+**Changing User Security**
 
-  Return the number of owner’s tokens allowed to be spent by spender:
+```ts
+it('should construct changeSecurity args properly', () => {
```
Comment (gyroflaw) [inline client-js/README.md:197]: please remove this line :)

**Review by gyroflaw — CHANGES_REQUESTED** [review]
Looks good, a minor change required

File: client-js/README.md (hunk 1 above)
Comment (ipopescu) [reply client-js/README.md]: Done, thanks for catching this!

File: client-js/README.md [hunk 2]
```diff
@@ -6,140 +6,219 @@ This JavaScript client gives you an easy way to install and interact with the Ca
...
-  const allowance = await cep18.allowances(
-    ownersPublicKey,
-    spenderPublicKey
-  );
-  ```
 
 ## Event Handling
 
-CEP18 token supports [CES](https://github.com/make-software/casper-event-standard) and the token can be intalled with CES or NoCES method. If the token is installed with CES method, you can listen token events by using `EventStream` from `casper-js-sdk`. To consume token events, you should install `@make-software/ces-js-parser` by running
+The CEP-18 token supports the [Casper Event Standard (CES)](https://github.com/make-software/casper-event-standard), and the token can be installed with or without event logging as described [here](../cep18/README.md#eventsmode). If you install the token with the EventsMode set to CES, you can listen to token events using the `EventStream` from the `casper-js-sdk`. To consume token events, you should also install the `@make-software/ces-js-parser` by running this command:
```
Comment (ACStoneCL) [inline client-js/README.md:215]:
```suggestion
CEP-18 tokens support the [Casper Event Standard (CES)](https://github.com/make-software/casper-event-standard), and tokens can be installed with or without event logging as described [here](../cep18/README.md#eventsmode). If you install a token with the EventsMode set to CES, you can listen to token events using the `EventStream` from the `casper-js-sdk`. To consume token events, you should also install the `@make-software/ces-js-parser` by running this command:
```

Something about "The CEP-18 token supports the Casper Event Standard" feels off to me, and I can't quite put my finger on it.

**Review by ACStoneCL — COMMENTED** [review]
One small nit that you can probably safely ignore, otherwise LGTM and I will leave approval for tech side.

**Review by gyroflaw — APPROVED** [review]
LGTM 👍

**Comment by ipopescu** [comment]
Thanks, @gyroflaw and @ACStoneCL!

**Pull request merged**